capstone-backend/app.js

```diff
@@ -284,30 +334,30 @@ app.get('/user/interests', async(req, res) => {
 
         movieQuery.equalTo("User", currentUser);
```

Comment:
All the the queries looks very similar, it would be good to refactored it out and pass in the type. As mentioned above, better for when we need to add more types :)